src/editors/vim.ts

```diff
@@ -1,24 +1,24 @@
-import "./editor";
+import Editor from "./editor";
 import { CommandExists } from "../lib/command-exists";
 
-export default class Vim implements Editor {
+export default class Vim extends Editor {
   private commandExists = new CommandExists();
 
   public get name(): string {
-    return "vim";
-  }
-
-  public get displayName(): string {
     return "Vim";
   }
 
   public get icon(): string {
     return "";
   }
 
+  public get binary(): string {
+    return "vim";
```

Comment:
Let's return an array of strings here, to support editors with multiple possible binary names:

```
public get binary(): string[] {
    return ['vi', vim'];
}
```

Reply:
done